server/routes/upload.js

```diff
@@ -10,6 +10,9 @@ const rule = {
     entityType: 'required|in:room,event',
     entity_id: 'required|numeric|min:1'
 }
+const coverRule = {
```

Comment:
hardcode noooooo )
const tepes = Object.keys (constants.rating.entity_types);
in: types.join();
or use lodash, if I do not mistake func .values

Reply:
done